rotkehlchen/api/rest.py

```diff
@@ -3196,3 +3196,61 @@ def purge_gitcoin_grant_data(self, grant_id: Optional[int]) -> Response:
             self.rotkehlchen.data.db, self.rotkehlchen.msg_aggregator,
         ).delete_gitcoin_ledger_actions(grant_id)
         return api_response(OK_RESULT, status_code=HTTPStatus.OK)
+
+    def add_manual_price(  # pylint: disable=no-self-use
+        self,
+        from_asset: Asset,
+        to_asset: Asset,
+        price: Price,
+        timestamp: Timestamp,
+    ) -> Response:
+        historical_price = HistoricalPrice(
+            from_asset=from_asset,
+            to_asset=to_asset,
+            source=HistoricalPriceOracle.MANUAL,
+            timestamp=timestamp,
+            price=price,
+        )
+        GlobalDBHandler().add_single_historical_price(historical_price)
+        return api_response(OK_RESULT, status_code=HTTPStatus.OK)
```

Comment:
When you implement what I asked in `add_single_historical_price` you can now here return more information and inform the caller if the addition worked or not. Also adjust api docs for the possible failure.